docs/examples/crewai_research_report.md

```diff
@@ -0,0 +1,45 @@
+# CrewAI Research Report
+
+The **CrewAI Research Report** is a minimal agentic system demonstrating how to connect an LLM to CrewAI using a custom-coded tool.
+
+---
+
+## File
+
+[crewai_research_report.hocon](../../registries/crewai_research_report.hocon)
+
+---
+
+## Prerequisites
+
+- This agent is **disabled by default**. To test it:
+  ```bash
+  pip install crewai
+  ```
```

Comment:
Add:
And edit `registries/manifest.hocon` to set `"crewai_research_report.hocon": false,` to `true`